README.md

```diff
@@ -124,6 +121,9 @@ try {
 * [AWS SDK Bundle for Symfony][mod-symfony]
 * [Amazon SNS Message Validator for PHP][sns-validator] - SNS validator without requiring SDK
 * [Guzzle Version 6][guzzle-docs] – PHP HTTP client and framework
+* For **the previous** version 2 of the SDK:
```

Comment:
This should either be 'For the previous version of the SDK:' or 'For Version 2 of the SDK:', as the current is grammatically incorrect. I'd prefer the latter in this case.